test/controllers/actions/AuthActionSpec.scala

```diff
@@ -77,14 +77,78 @@ class AuthActionSpec extends ControllerSpecBase {
 
       withAuthError(InsufficientEnrolments("")) {
 
-        val response = authController.action(FakeRequest())
+        val response = authController().action(FakeRequest())
 
         status(response) mustBe SEE_OTHER
         redirectLocation(response) mustBe Some(controllers.routes.UnauthorisedController.onPageLoad.url)
       }
     }
   }
 
+  "Auth action" should {
+
+    "allow access" when {
+
+      "whitelisting doesn't have any eori and user has eori" in {
```

Comment:
i see it is correct!

Reply:
Actually, i should have said, "I see you meant to do it like that!"  :-)

I guess once a service is "live" we remove the whitelist and then it's free-for-all.

Reply:
yeah, that's true :)